src/webrtc/callFeed.ts

```diff
@@ -39,6 +39,8 @@ export interface ICallFeedOpts {
      * Whether or not the remote SDPStreamMetadata says video is muted
      */
     videoMuted: boolean;
+
+    setVADMute?: (muted: boolean) => void;
```

Comment:
This might be obvious but what does VAD mean/stand for?

Reply:
Voice Activation Detection

We can change it, or perhaps just make a comment on it?